ext/byebug/byebug.h

```diff
@@ -125,7 +125,7 @@ extern VALUE next_thread;
 
 /* functions from context.c */
 extern void Init_context(VALUE mByebug);
```

Comment:
You need to replace this function as well, right?

Reply:
Thank you for your review. I fixed it.